lib/elasticsearch.js

```diff
@@ -84,7 +84,8 @@ const $elastic = function(){
             objSearch.body.aggs = {
                 "restfulize_group_total": {
                     "cardinality": {
-                        "field": fieldName
+                        "field": fieldName,
+                        "precision_threshold": 10000
```

Comment:
I'm ok with this change but this number should really be driven by the total number of items in the index. We can keep an eye on three memory usage to see how this affects the performance of the ES cluster. @jasonfutch thoughts?